### ISSUE TITLE
Question Regarding End Model

### ISSUE THREAD

**Issue by tschamp31** [weinman/cnn_lstm_ctc_ocr#66]
Preface: So my background when coming into machine learning is mainly just understanding programming. Ive gained some solid knowledge about data framing etc but the math of neural networks is still a major weakness.

So my question I believe I already understand but I want to verify before I waste time on these efforts. 
1.) The MJSYNTH dataset will only teach the model how to break down/identify/process singular wards? 

2.) Assuming yes to (2); Which in result means it needs to be taught how to read sentence and paragraph/spacing structures, correct?

3.) Assuming yes to (2); Is that where your teams work on mapsynth came into play? 

4.) Assuming yes to (3); Is the finished mjsynth model then trained on the mapsynth dataset or is it finetuned/scoped to the mapsynth dataset? 

5.) Assuming yes to (4); What is the global_step/loss/learning rate etc that is ideal to train it on that dataset? 

Yes/No should suffice for all 5. If no maybe a very short rationale. Thank you again for this project being public. Wish your team well on the 2019 ICDAR. I will also post a 1 million step model trained on a single GPU if your team would like a copy on hand or provide to the public.
